### ISSUE TITLE
Should this be used anymore?

### ISSUE THREAD

**Issue by adamralph** [probot/stale#349]
I noticed that the app page is a 404 - https://probot.github.io/apps/stale/

And it seems that we should now be using https://github.com/actions/stale instead?

**Comment by hanistiflerrr** [comment]
Yes do blind that plz send me your addres in mojaz on undergrand game shows today for ever . Thats my party yes yes

**Comment by hanistiflerrr** [comment]
Showsparty*

**Comment by adamralph** [comment]
reported as spam

**Comment by adamralph** [comment]
@seeflood I assume we should now use https://github.com/actions/stale

**Comment by WoLpH** [comment]
For me personally the stale bot is still doing something but I would not recommend using it. It seems to run every few months or so and when it does it immediately gives me a large batch of stale issues.

In any case, it's definitely not reliable enough

**Comment by poloka** [comment]
I do wish this project would remain in usage because not all enterprise organization allow/enable usage of Actions within their GitHub instances.

**Comment by FranklinYu** [comment]
The link was removed in probot/probot.github.io#376. It’s still in the marketplace, though: https://github.com/marketplace/stale

**Comment by hanistiflerrr** [comment]
For use wht . Work for onit guys for mm beggos

**Comment by WoLpH** [comment]
> For use wht . Work for onit guys for mm beggos

Did your cat walk across the keyboard or are you trying to say something?

**Comment by FranklinYu** [comment]
> Did your cat walk across the keyboard or are you trying to say something?

Look up, the user has been speaking lorem ipsum. It must be either a cat or a robot.

**Comment by WoLpH** [comment]
> > Did your cat walk across the keyboard or are you trying to say something?
> 
> Look up, the user has been speaking lorem ipsum. It must be either a cat or a robot.

You're right. I didn't notice the earlier messages because they were hidden. I've reported the account to Github
